distribution/install.sh

```diff
@@ -168,7 +168,7 @@ install_from_archive() {
     printf " ✓\n"
 
     if [ "$modify_path" = "yes" ]; then
-      local _path="export PATH=\"\$HOME/.vector/bin:\$PATH\""
+      local _path='export PATH="$PATH:$HOME/.vector/bin"'
```

Comment:
I think we want to maintain the escaping here since this is the string that is added to `.zprofile` or `.profile`.

Reply:
Actually, the escaping here should be equivalent. All the meta-characters are escaped in the removed line.

Reply:
The quoting might have been due to shellcheck

Reply:
Ah, doh, I missed that `"` was swapped for `'`.

Reply:
It does look like shellcheck (run in CI) isn't a fan though:

```
      local _path='export PATH="$PATH:$HOME/.vector/bin"'
                  ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.
```

I think we could ignore the rule in this particular circumstance.

Reply:
Yeah, I think we are ok intentionally _not_ expanding there.

Reply:
I've added a shellcheck ignore for 2016